### PR TITLE
Push bugfix into 21.0.0

### PR DESCRIPTION
**Pull request by ridz1208**
This pulls the latest bugfix changes into the 21.0-release branch.

Note that there were conflicts in php/libraries/User.class.inc and modules/dataquery/jsx/reacts.tabs.js. Extra care should be taken reviewing file changes from those files.

### CODE REVIEW

**Comment by driusan** [comment]
approvals were on #4697, so I'm just merging this once it passes.

**Comment by kongtiaowang** [comment]
/home/travis/build/aces/Loris/modules/issue_tracker/jsx/index.js
       7:3   error    Unexpected var, use let or const instead       no-var
       7:32  error    Strings must use singlequote                   quotes

File: modules/issue_tracker/jsx/index.js [hunk 1]
```diff
@@ -4,12 +4,12 @@ import IssueForm from './IssueForm';
  * Render IssueForm on page load
  */
 $(function() {
-  const args = QueryString.get();
+  var id = location.href.split("/issue/")[1];
```
Comment (kongtiaowang) [inline modules/issue_tracker/jsx/index.js]: const instead fo var

Comment (ridz1208) [reply modules/issue_tracker/jsx/index.js]: @HenriRabalais does that looks like it makes sense to you Mr issue tracker guy ?

Comment (maltheism) [reply modules/issue_tracker/jsx/index.js]: @ridz1208 var isn't supposed to be used anymore. The two options are either const or let and with const being what should be used in this case like @kongtiaowang suggested.

**Pull request merged**